defaults/Debian.yml

```diff
@@ -1,7 +1,5 @@
-U---
+---
 # See https://wkhtmltopdf.org/downloads.html for checksums.
```

Comment:
Please move `package_source`, `download_dest`, and `package_checksum` variables, to main.yml, in the same style that `xfonts_package_name` has been adjusted. Unfortunately there's no such thing as "include"s for defaults. It has to all happen inside main.yml.

Once they're working in main.yml, this file can be removed.

Reply:
Done.